lib/IRGen/GenClangDecl.cpp

```diff
@@ -100,7 +113,7 @@ void IRGenModule::emitClangDecl(const clang::Decl *decl) {
   SmallVector<const clang::Decl *, 8> stack;
   stack.push_back(decl);
 
-  ClangDeclFinder refFinder([&](const clang::Decl *D) {
+  auto callback = [&](const clang::Decl *D) {
     for (auto *DC = D->getDeclContext();; DC = DC->getParent()) {
```

Comment:
I believe this is where I was encountering the operator() for the std::functional that wasn't getting added to the stack.